components/nexus-repository/src/main/java/org/sonatype/nexus/repository/storage/DocumentWrapperSupport.java

```diff
@@ -16,35 +16,36 @@
 
 import javax.annotation.Nullable;
 
+import com.google.common.collect.ImmutableSet;
 import com.orientechnologies.orient.core.id.ORID;
-import com.tinkerpop.blueprints.impls.orient.OrientVertex;
+import com.orientechnologies.orient.core.record.impl.ODocument;
 import org.joda.time.DateTime;
 
 import static com.google.common.base.Preconditions.checkNotNull;
 import static com.google.common.base.Preconditions.checkState;
 
 /**
- * Supporting base class for {@link VertexWrapper} implementations.
+ * Supporting base class for {@link DocumentWrapper} implementations.
  *
  * @since 3.0
  */
-abstract class VertexWrapperSupport
-    implements VertexWrapper
+abstract class DocumentWrapperSupport
+    implements DocumentWrapper
```

Comment:
I guess though this was the change I was expecting to see if we are converting to the Document api, why not use the framework we already built to deal with it vs. another framework just for this?